src/ztc/api/serializers/zaken.py

```diff
@@ -262,8 +268,23 @@ class Meta:
             'einde_geldigheid': {
                 'source': 'datum_einde_geldigheid'
             },
+            'draft': {
+                'read_only': True,
+            },
         }
 
         # expandable_fields = {
         #     'catalogus': ('ztc.api.serializers.CatalogusSerializer', {'source': 'catalogus'}),
         # }
+
+    @transaction.atomic()
+    def create(self, validated_data):
+        zaaktypen_relaties_data = validated_data.pop('zaaktypenrelaties', None)
```

Comment:
I don't see this field defined anywhere, am I missing something?

Reply:
this is source for `gerelateerde_zaaktypen` field (which is a nested serializer):
https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus/blob/906edb55968e5fb61d98c61cadc143e61a829be3/src/ztc/api/serializers/zaken.py#L128